Makefile

```diff
@@ -47,3 +47,14 @@ docker-serve:
 test-examples:
 	scripts/test_examples.sh install
 	scripts/test_examples.sh run
+
+link-checker-setup:
+	curl https://raw.githubusercontent.com/wjdp/htmltest/master/godownloader.sh | bash
+
```

Comment:
Is this setup similar to:
https://github.com/linkerd/website/blob/master/Makefile

Reply:
OK. I caught up with this topic on Slack.